app/routes/public/speakers.js

```diff
@@ -3,36 +3,40 @@ import Route from '@ember/routing/route';
 export default Route.extend({
   async model() {
     const eventDetails = this.modelFor('public');
-    return {
-      event    : eventDetails,
-      speakers : await eventDetails.query('speakers', {
-        filter: [
+    let filterOptions = [];
```

Comment:
Why is it let?

Reply:
Changing to const !

Reply:
@iamareebjamal Done

Reply:
Changing to const will fail the ESLintrule as filterOption may tend to change , hence let should be used

Reply:
> Changing to const will fail the ESLintrule

Then that ESLintrule is stupid. Nowhere in this code filterOptions is changed, neither can this reference ever be changed

Change it to const